README.md

```diff
@@ -11,74 +11,66 @@ to build on.
 
 ## Features
 
+* Allows for submitting requests and testing responses
 * Maintains a cookie jar across requests
-* Easily follow redirects when desired
-* Set request headers to be used by all subsequent requests
-
-## Supported platforms
-
-* Ruby 2.0-3.1
-* JRuby 9.1.+
-
-If you are using and older version of Ruby, use rack-test 0.6.3.
-
-## Known incompatibilites
-
-* `rack-test >= 0.71` _does not_ work with older Capybara versions (`< 2.17`). See [#214](https://github.com/rack/rack-test/issues/214) for more details.
+* Supports request headers used for subsequent requests
+* Follow redirects when requested
 
 ## Examples
 
-(The examples use `Test::Unit` but it's equally possible to use `rack-test` with other testing frameworks like `rspec`.)
+These examples use `test/unit` but it's equally possible to use `rack-test` with
+other testing frameworks such as `minitest` or `rspec`.
 
 ```ruby
 require "test/unit"
 require "rack/test"
+require "json"
 
 class HomepageTest < Test::Unit::TestCase
   include Rack::Test::Methods
 
   def app
-    app = lambda { |env| [200, {'Content-Type' => 'text/plain'}, ['All responses are OK']] }
-    builder = Rack::Builder.new
-    builder.run app
+    lambda { |env| [200, {'Content-Type' => 'text/plain'}, ['All responses are OK']] }
   end
 
   def test_response_is_ok
-    get '/'
-
-    assert last_response.ok?
-    assert_equal last_response.body, 'All responses are OK'
-  end
+    # Optionally set headers used for all requests in this spec:
+    #header 'Accept-Charset', 'utf-8'
```

Comment:
Would advise we show using lower case headers if possible.

Reply:
Certainly possible.  I think this isn't the only case where they are used, so how about a separate commit to fix all cases later?